File: src/androidTest/java/pt/joaomneto/titancompanion/TestTPOP.java
package pt.joaomneto.titancompanion;


import android.support.test.filters.LargeTest;
import android.support.test.runner.AndroidJUnit4;

import org.junit.runner.RunWith;

import pt.joaomneto.titancompanion.TestTFOD;
import pt.joaomneto.titancompanion.consts.FightingFantasyGamebook;


@LargeTest
@RunWith(AndroidJUnit4.class)
public class TestTPOP extends TestTFOD {

    @Override
    protected FightingFantasyGamebook getGamebook() {
        return FightingFantasyGamebook.THE_PORT_OF_PERIL;
    }


}

File: src/androidTest/java/pt/joaomneto/titancompanion/TestTPOP.kt
package pt.joaomneto.titancompanion

import android.support.test.filters.LargeTest
import android.support.test.runner.AndroidJUnit4
import org.junit.runner.RunWith
import pt.joaomneto.titancompanion.consts.FightingFantasyGamebook

@LargeTest
@RunWith(AndroidJUnit4::class)
class TestTPOP : TestTFOD() {

    override val gamebook = FightingFantasyGamebook.THE_PORT_OF_PERIL


}